modules/sqlite_helpers.py

```diff
@@ -48,45 +48,47 @@ def insert_url(sqlite_file: str, url: str, alias: str):
         logger.exception("Inserting url had an error")
         return False
 
-def get_urls(sqlite_file: str): #returns all urls in the table
+def get_urls(sqlite_file, page, limit):
     db = sqlite3.connect(sqlite_file)
     cursor = db.cursor()
     
-    sql = "SELECT * from urls order by created_at desc"
-    cursor.execute(sql)
+    offset = (page - 1) * limit
+    sql = "SELECT * FROM urls LIMIT ? OFFSET ?"
+    cursor.execute(sql, (limit, offset))
     result = cursor.fetchall()
     url_array = []
     for row in result:
-        try:
-            url_data = {
+        url_data = {
                 "id": row[0],
                 "url": row[1],
                 "alias": row[2],
                 "created_at": row[3]
             }
-            url_array.append(url_data)
-        except KeyError:
-            continue
+        url_array.append(url_data)
     return url_array
 
-def get_url(sqlite_file: str, alias: str): #return the string for url entry for a specified alias
```

Comment:
keep this function as is, just update `get_urls` to have a new params like

```py
def get_urls(
  sqlite_file,
  page=0,
  limit=ROWS_PER_PAGE,
  search=None,
):
  # ...
```

Reply:
done